packages/node_modules/@node-red/editor-client/src/js/ui/palette.js

```diff
@@ -593,6 +593,11 @@ RED.palette = (function() {
         setTimeout(function() { $(window).trigger("resize"); } ,200);
     }
 
+    function changeSubflowColor(sf, color) {
```

Comment:
The `refreshNodeTypes` function (exposed as `RED.palette.refresh()`) is already used to update the appearance of subflows in the palette - such as when the name changes or the input/output port counts change. And it is already called at the end of the subflow edit process - see line 2322 of editor.js above.

Instead of adding `RED.palette.changeSubflowColor()`, `refreshNodeTypes` should update the colour of the node in the palette.

Reply:
Thank you!
Fixed to update subflow color in `refreshNodeTypes`.
Also fixed wrong subflow color specification in new code.